frontend/components/top_nav/SiteTopNav/SiteTopNav.tsx

```diff
@@ -174,13 +158,6 @@ const SiteTopNav = ({
       );
     }
 
-    if (
```

Comment:
@gillespi314 would appreciate your eyes on this change. It seems to allow the desired behavior (see [commit message for this commit](https://github.com/fleetdm/fleet/pull/21885/commits/d40126d1e74fec111c13b5239f03358f893c8862)), but it's not clear to me how it does. The other page this would seemingly affect, Queries, still works as expected.